README.md

```diff
@@ -34,21 +56,106 @@ Create a json config file with `branch-config` and/or `wctflags` keys:
 `tattoo_config.json`:
 ```
 {
-  "branch-config": {
-    "polymer" : "Polymer/polymer#lazy-register-extends",
-    "paper-button" : "PolymerElements/paper-button#master"
-  },
-  "wctflags": ["-b", "canary"]
+  "test": [
+    "PolymerElements/paper-button#2.0-preview"
+  ],
+  "wctflags": ["--local", "canary"]
 }
-
 ```
+
 Then run:
+
 ```
 tattoo
 ```
 
+Config files support most of the same options as the command-line flags:
 
-***Fix broken installation***
+* `"test": ["PolymerElements/paper-button#2.0-preview", "PolymerElements/*", etc]`
+  Repositories to test.
+* `"skip-test": ["PolymerElements/iron-meta", "*/*-alpha", etc]`
+  Repositories not to test.  Filters out items from the `test` list.
```

Comment:
I inherited some notion of these args when refactoring.  They have distinct purposes, but I'm questioning whether it really makes sense to have a separate `skip-test` argument, since its purpose is to work identically to `exclude-repo` except only effect the `test` collection.  At least that is how it works right now.

After thinking about it -- it is more sensible for "--skip-test" it to *just* skip the test for a repo and not exclude it from download and that "--exclude-repo" should drive what is cloned or not.

Follow up thought, I'd like to rename the `--repo` and `--exclude-repo` flags to `--require` and `--exclude`.  I'll keep `--skip-test` since its more intention revealing.  Adding issues for these changes.  #26 and #27